### ISSUE TITLE
Tooltip for buttons

### ISSUE THREAD

**Issue by prophet1996** [lapce/lapce#1171]
## Is your feature request related to a problem? Please describe.
I want to be able to see a tooltip if there's a binding attached to button/action it should have keymap string as an inlay or hint 

A tool tip attached to each button which appears on hover

**Comment by FancyBanana** [comment]
Coming from VSCode, this is one of the first things that I've noticed. This feature is important for someone who's just started using the editor and does not know any keybindings`

**Comment by ghenry** [comment]
Yep, I just thought of exactly this. Presuming tooltips are support in the UI framework and we could add these ourselves and submit a PR?
